### ISSUE TITLE
Make the Storage Bus Significantly Cheaper

### ISSUE THREAD

**Issue by toaster128** [Nomi-CEu/Nomi-CEu#1123]
### Nomi CEu Version

1.7.3

### Launcher

ATLauncher

### Configurations or Mods Changed

Added ZBGT

### Environment

Singleplayer

### Mode

Both Modes

### Related Problem

It is incredibly frustrating to not be able to easily move from chests to using AE2 storage. Interfaces are rather expensive in early MV, making it quite difficult to place storage buses on pre-existing storage, such as chests. 

### Your Solution

I would make the ME Storage Bus cost a Fluix Plate and an MV Pump. This makes it significantly cheaper, and easier to use.
I would also make the ME Fluid Storage Bus cost a Fluid Regulator and a Basic Buffer, which would make it cheaper while still keeping it at a higher price due to being more useful than the normal Storage Bus (as fluid cells are less good compared to item cells).

### How Will Your Solution Work

This solution makes it easier to go from using early-game storage by making the transition to AE2 storage less expensive in circuits and in Aluminum.

### Additional Information

ME Interface currently costs 5 Aluminum Plates, an MV Robot Arm, a MV Machine Hull, a Formation Core, and an Annihilation core.
ME Storage Bus currently costs 1 ME Interface and 1 MV Electric Piston.
My proposal would decrease this cost significantly. I do not believe applying the same change to Fluid Storage Buses would be similarly useful, and thus my proposed change to it will be less immpactful.

**Comment by SilverSoggy** [comment]
Use drives pal. And please, keep in kind that it is a greg tech modpack.

**Comment by D-Alessian** [comment]
I don't agree with the way the message above approached this, but I will close this as it really just isn't expensive. I would tell anyone facing this issue to just, start working on making autocrafting, and make sure interfaces are autocraftable.

You can also use denser storage if you want to kick start AE2 with current infra, like drawers with drawer controllers, or using Actually Additions crates.

And fluid cells are considerably better than item cells, given the comparative options, this was wrong as well.
